run_websocket_server.py

```diff
@@ -152,7 +152,8 @@ def target():
     parser.add_argument(
         "--testing",
         action="store_true",
-        help="if set, websocket server worker will load the test dataset instead of the training dataset",
+        help="""if set, websocket server worker will load
+                the test dataset instead of the training dataset""",
```

Comment:
I guess flake8 should adapt to our black setting for line length?

Reply:
We had a 100 characters limit. We can increase that in ```.flake8```. Should we add more characters to that limit?

Reply:
This line had a bigger character number, but it might not be detected because it was introduced before flake8/black was introduced?